### PR TITLE
feature : add API send all crypto in BEP20 network

### PR DESCRIPTION
**Pull request by hichri-louay**

### CODE REVIEW

File: libs/transfer/transfer-tokens.js [hunk 1]
```diff
@@ -10,7 +10,9 @@ module.exports.transferTokens = async function ({
     provider,
     walletPassword,
     encryptedPrivateKey,
+    token = false,
 }) {
+    console.log(token)
```
Comment (HamdiBenK) [inline libs/transfer/transfer-tokens.js]: remove console.log please

File: libs/transfer/transfer-tokens.js [hunk 2]
```diff
@@ -56,6 +58,14 @@ module.exports.transferTokens = async function ({
             tokenSmartContractAddress === null ||
             tokenSmartContractAddress === process.env.TOKEN_BTT_CONTRACT
         ) {
+            console.log(1111111111111, amount)
```
Comment (HamdiBenK) [inline libs/transfer/transfer-tokens.js]: remove console

File: libs/transfer/transfer-tokens.js [hunk 3]
```diff
@@ -64,6 +74,8 @@ module.exports.transferTokens = async function ({
                 gasPrice,
             })
         } else {
+            console.log(2222222222222222222, toAddress)
```
Comment (HamdiBenK) [inline libs/transfer/transfer-tokens.js]: remove console

Comment (HamdiBenK) [inline libs/transfer/transfer-tokens.js]: remove console

Comment (hichri-louay) [reply libs/transfer/transfer-tokens.js]: done

Comment (hichri-louay) [reply libs/transfer/transfer-tokens.js]: done

File: libs/transfer/transfer-tokens.js (hunk 2 above)
Comment (hichri-louay) [reply libs/transfer/transfer-tokens.js]: done

File: libs/transfer/transfer-tokens.js (hunk 1 above)
Comment (hichri-louay) [reply libs/transfer/transfer-tokens.js]: done

**Pull request merged**